spec/controllers/ops_controller/settings/common_spec.rb

```diff
@@ -231,26 +231,6 @@
       end
     end
 
-    context "#restore_password" do
```

Comment:
The `restore_password` method was only used for replication and was also removed in this PR (https://github.com/ManageIQ/manageiq/pull/9696/files#diff-84dbfcd1a7ac9f7442c66dadf41485bbL1245) so I think this is safe to remove